aws/DRVault-Single-Deployment.yaml

```diff
@@ -411,7 +427,7 @@ Resources:
               cfnresponse.send(event, context, cfnresponse.SUCCESS, {}, id)
             except:
               traceback.print_last()
-              cfnresponse.send(event, context, cfnresponse.FAIL, {}, "ok")
+              cfnresponse.send(event, context, cfnresponse.FAIL, {}, "ami not found")
```

Comment:
does `ami not found` is the only way to get here? what if there is a network issue... wouldn't this cause the customer to see `ami not found` ?

Reply:
changed error message